pkg/loadbalancer/experimental/testdata/svc-type-annotation.txtar

```diff
@@ -0,0 +1,262 @@
+#! --enable-experimental-lb --lb-test-fault-probability=0.0
+
+# Add some node addresses
```

Comment:
Could maybe simplify the tests by dropping the node addresses as they aren't relevant here?